modules/complex-number/test/test_yury_gaydaychuk_complex_number.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 Yury Gaydaychuk
+
+#include <gtest/gtest.h>
+
+#include "include/complex_number.h"
+
+// Multiplication on 1 must be precise
+TEST(Gaydaychuk_Yury_ComplexNumberTest, Precise_One_Multiplication) {
+    // Arrange
+    ComplexNumber z1(1.0, 0.0);
+    ComplexNumber z2(22.33, 22.33);
+
+    // Act
+    ComplexNumber z3 = z1 * z2;
+
+    // Assert
+    EXPECT_EQ(z2, z3);
```

Comment:
Я не уверен, что это всегда правда
```suggestion
    EXPECT_FLOAT_EQ(z2, z3);
```

Reply:
Во всех трех случаях проверяются базовые алгебраические равенства. Если они не выполняются точно, то алгоритм reimplemented (перемудрен).